### DISCUSSION TITLE
How do I use folders like /after/ftplugin with aniseed?

### DISCUSSION THREAD

**Discussion by katawful** [Olical/aniseed#55] [Q&A]
I have a bunch of filetype plugins in /after/ftplugin and I can't find a way to compile lua to these folders using aniseed

**Comment by Olical** [comment]
Not sure why you need to compile Lua to that location since it should all be executable under the `lua` directory, but this may help? https://github.com/Olical/aniseed/blob/4ca3d418eebc0da452b7defc18970c83f7de5070/doc/aniseed.txt#L211-L218

I don't know if you're writing a plugin or managing your dotfiles though, both have options to output the compiled results under a different tree I think. You won't be able to selectively compile some files into one dir and another into a different one.

**Reply by katawful** [reply]
I'm using it for dotfiles yes

If I put the /after/ and similar folders (written in fennel of course) into /nvim/fnl it just doesn't load the files once compiled to /nvim/lua

**Reply by Olical** [reply]
Ah okay, so maybe it's a misunderstanding. The files under `lua` are _not_ loaded and executed automatically for you, only `fnl/init.fnl` is loaded automatically.

See my dotfiles for an example configuration were I load a different module

https://github.com/Olical/dotfiles/blob/5a0faf9ac61ae513610253ffb6e55beb87d9eda1/stowed/.config/nvim/init.lua#L24-L28

Then see the module itself to see how I cascade into the other modules by requiring them

https://github.com/Olical/dotfiles/blob/5a0faf9ac61ae513610253ffb6e55beb87d9eda1/stowed/.config/nvim/fnl/dotfiles/init.fnl

I hope that helps!

**Reply by katawful** [reply]
Will a folder like /after/ftplugin still function like it would in just vimscript? I prefer loading filteype configs after all plugins load in case something gets modified
